mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/KitPvp.java

```diff
@@ -268,9 +268,7 @@ public KitOwnership removeKit(Transaction transaction, Kit kit) {
         Set<Kit> kits = transaction.getProperty(DSLContext.class)
                 .select(KITPVP_KITS_OWNERSHIP.KIT_ID).from(KITPVP_KITS_OWNERSHIP)
                 .where(KITPVP_KITS_OWNERSHIP.USER_ID.eq(this.userId))
-                .fetchSet((ownershipRecord) -> {
-                    return manager.getKit(transaction, ownershipRecord.get(KITPVP_KITS_OWNERSHIP.KIT_ID));
-                });
+                .fetchSet((ownershipRecord) -> manager.getKitById(transaction, ownershipRecord.get(KITPVP_KITS_OWNERSHIP.KIT_ID)));
```

Comment:
Don't reformat existing code unless it's necessary. Just change `getKey` to `getKitById`.

Don't accept IntelliJ's suggestions to remove brackets ({}) from lambdas. Disable that suggestion. If you want to avoid messing with the default settings, you can create another inspection profile in Intellij, edit that inspection profile, and use that profile for SolarMC.

Reply:
oh ok...